composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "dpc-sdp/tide_core": "^1.2.5",
-        "dpc-sdp/tide_media": "^1.2.5"
+        "dpc-sdp/tide_media": "^1.2.5",

```

Comment:
Latest version of tide core and tide media is 1.2.6.

Reply:
solved.